src/packet/mod.rs

```diff
@@ -1,6 +1,77 @@
 //! # Packet module
 //!
 //! This module handles everything in relationship to packets.
+//!
+//! ```rust
+//! # const DATA :&'static [u8] = &[0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08];
```

Comment:
might be nicer to do sth like `const DATA: &[u8] = b"hello world"` instead of numbers